client/my-sites/checkout/checkout-thank-you-i2/data/use-thank-you-data.tsx

```diff
@@ -0,0 +1,10 @@
+import { translate } from 'i18n-calypso';
```

Comment:
Some test data, not sure how we should extend this yet, but I think this approach is nice as we try our best to separate data away from the component.

Reply:
Is this coming from an api? If not we can skip a few steps and mark it up with JSX directly:

```
export function PlanThankYouHeading() {
    return (
        <ThankYouTitle>{ translate( 'Get the best out of your site' ) }</ThankYouTitle>
        <ThankYouSubTitle>{ translate( 'All set! Start exploring the features included with your [plan name] plan' ) }</ThankYouSubtitle>
    );
}
```